packages/plugin-ext/src/common/plugin-protocol.ts

```diff
@@ -57,8 +58,18 @@ export interface PluginPackage {
     extensionPack?: string[];
 }
 export namespace PluginPackage {
+    function encodeFileURIComponent(value: string): string {
+        return encodeURIComponent(value).replace(/%(2f|5c)/gi, '/');
```

Comment:
What for `/%(2f|5c)/gi`?

Reply:
We want to encode everything that needs to be encoded (which `encodeURIComponent` does) except slashes, which should be normalized to forward slashes, which the regex replace does.